test/support/commands.js

```diff
@@ -19,6 +25,8 @@ Cypress.Commands.add('showPreview', (token, fileId, options) => {
         cy.window().then((win) => {
             win.loadPreview(options);
         });
+    } else {
+        cy.getByTestId('load-preview').click();
```

Comment:
Is this needed or can we just remove the `if (options)` check above and allow `win.loadPreview(options)` to be used for all `showPreview` invocations. It seems like it would be the same effect.

Reply:
Yeah that's cleaner let me change that